fbw-a380x/src/systems/instruments/src/ND/OansControlPanel.tsx

```diff
@@ -649,7 +642,10 @@ export class OansControlPanel extends DisplayComponent<OansProps> {
                         BTV MANUAL CONTROL / FALLBACK
                       </div>
                     </div>
-                    <div class="oans-cp-map-data-btv-rwy-length">
+                    <div
+                      class="oans-cp-map-data-btv-rwy-length"
+                      style={{ visibility: this.fmsDataStore.landingRunway.map((rwy) => (rwy ? 'inherit' : 'hidden')) }}
```

Comment:
Make this MappedSubject a class prop rather than 3 instances of the exact same subject.

Reply:
Makes sense, fixed in next commit